src/bse202/templates/views/login.html

```diff
@@ -33,15 +33,29 @@ <h1>Login</h1>
                 <input type="password" id="password" name="password" required><br>
                 <button type="submit" class="btn">Submit</button>
                 <div class="secondary-buttons">
-                    <button type="button" class="btn-secondary">Create Account</button>
+                    <button type="button" class="btn-secondary" onclick="gotosignup()">Create Account</button>
                     <button type="button" class="btn-secondary">Forgot Password</button>
                 </div>
             </form>
-            {% if error is defined %}
-                <p>{{ error['message'] }}</p>
+            {% if message %}
+                <p>{{ message }}</p>
```

Comment:
will we no longer be using the actual sign-up page?
If so i'll remove the GET handler for it

Reply:
> [!NOTE]
> sorry, wrong lines, right file though

Reply:
im not sure that was halims other contribution